packages/redux/src/recommendations/__tests__/reducer.test.ts

```diff
@@ -48,19 +50,21 @@ describe('recommendations reducer', () => {
     });
 
     it('should handle other actions by returning the previous state', () => {
-      const state = {
+      const state: State = {
         error: {
           [mockRecommendationsStrategy]: error,
         },
+        isLoading: {},
+        result: {},
       };
 
-      expect(reducer(state).error).toEqual(state.error);
+      expect(reducer(state, mockAction).error).toEqual(state.error);
     });
   });
 
   describe('isLoading() reducer', () => {
     it('should return the initial state', () => {
-      const state = reducer();
+      const state = reducer(initialState, mockAction);
```

Comment:
Same as above, initialState should be replaced with `undefined`. There are a few places in this file that contain the same pattern, can you review all of them and change where necessary?

Reply:
Source updated, can you check again.